src/app/feed/user-info-box/user-info-box.component.html

```diff
@@ -5,10 +5,10 @@
 	<div *ngIf="(!isUserResponseLoading)" class="wrapper">
 		<div class="image-gallery">
 			<div class="profile-banner-image">
-				<img class="banner" src="{{ apiResponseUser.profile_banner_url }}"/>
+				<img class="banner" src="{{ apiResponseUser.profile_banner_url }}" onError="this.src='./assets/images/default600x200.jpg';"/>
 			</div>
 			<div class="profile-image">
-				<img src="{{ apiResponseUser.profile_image_url }}"/>
+				<img src="{{ apiResponseUser.profile_image_url }}" onError="this.src='./assets/images/def65.jpg';"/>
```

Comment:
@praveenojha33 The default image gets load only when there is any error(like broken URL). I have not replaced the original response url. As mentioned in the PR description, the problem is with twitter image servers, which do not allow to access a lot of images and sometimes none. And it also depends on host response capacity.